manifests/params.pp

```diff
@@ -181,6 +185,7 @@
   $puppet_url                 = "https://${::fqdn}:8140"
   $puppet_use_environment_api = undef
   $puppet_use_cache           = undef
+  $puppet_envdir              = "${puppet_codedir}/environments"
```

Comment:
@mmoll Like this? This doesn't feel very DRY to duplicate all the logic we have in other modules here...